src/components/app/app.jsx

```diff
@@ -31,11 +31,14 @@ const App = () => {
           </MainPage>
         </Route>
         <Route exact path="/login">
-          <LoginPage>
-            <Header/>
-            <LoginForm/>
-            <Location/>
-          </LoginPage>
+          { isUserAuthorized ? <Redirect
+            to={`/`}
+          /> :
+            <LoginPage>
+              <Header/>
+              <LoginForm/>
+              <Location/>
+            </LoginPage>}
```

Comment:
FYI Могу предложить вот такой стиль. Но это не обязательно

```suggestion
          {
             isUserAuthorized
               ? <Redirect
                    to={`/`}
                  />
               : <LoginPage>
                  <Header/>
                  <LoginForm/>
                  <Location/>
                </LoginPage>
          }
```